src/components/Properties/PropertyMultipleText.vue

```diff
@@ -142,6 +142,7 @@ export default {
 	user-select: none;
 	text-align: right;
 	background-size: 16px;
+	padding-right: 7px;
```

Comment:
Good catch! Since this is very close to 8px, we could use the variable `calc(var(--default-grid-baseline) * 2)` which would come up to 8px, what do you think? :)

Reply:
done